moped-editor/src/components/Maps/GeocoderControl.js

```diff
@@ -13,7 +13,7 @@ const austinFullPurposeJurisdictionFeatureCollection = {
       name: "EPSG:4326",
     },
   },
-  bbox: [-97.940377, 30.133717, -97.578205, 30.464826],
+  bbox: [-98.182, 29.987, -97.304, 30.663],
```

Comment:
The reduction in precision is intentional. According to [this XKCD comic](https://xkcd.com/2170/), 3 decimal places is the scale of a small street, so that felt right.